packages/smart-wallet/src/invitations.js

```diff
@@ -70,11 +70,29 @@ export const makeInvitationsHelper = (
       assert(instance && description, 'missing instance or description');
       /** @type {Amount<'set'>} */
       const purseAmount = await E(invitationsPurse).getCurrentAmount();
-      const match = AmountMath.getValue(invitationBrand, purseAmount).find(
-        details =>
-          details.description === description && details.instance === instance,
+      const invitations = AmountMath.getValue(invitationBrand, purseAmount);
+      const matchingDescription = invitations.filter(
+        details => description === details.description,
       );
-      assert(match, `invitation not found: ${description}`);
+
+      let match;
+      if (matchingDescription.length === 0) {
+        assert.fail(`no invitation found: ${description}`);
+      } else if (matchingDescription.length === 1) {
+        match = matchingDescription[0];
+        if (instance !== match.instance) {
+          // FIXME(PS0) match strictly
+          console.warn(
+            `invitation search ignoring instance because one match for $(description) in purse`,
+          );
+        }
```

Comment:
This shouldn't be necessary since #6167.  Let me know if you're still seeing issues, and I can show you how to track down distinct instances.